test/integration/coscheduling_test.go

```diff
@@ -212,23 +320,43 @@ func TestCoschedulingPlugin(t *testing.T) {
 			t.Logf("Start-coscheduling-test %v", tt.name)
 			testutils.CleanupPods(cs, t, pods)
```

Comment:
(this is pre-existing)

```suggestion
			defer testutils.CleanupPods(cs, t, pods)
```